network-api/networkapi/wagtailpages/migrations/0047_auto_20221013_1743.py

```diff
@@ -0,0 +1,110 @@
+# Generated by Django 3.2.13 on 2022-10-13 17:43
```

Comment:
It'd be good to name this migration

Reply:
Updated to [0047_update_searchimage_label_and_help_text.py](https://github.com/mozilla/foundation.mozilla.org/blob/da8b6fb62a2bddab24913644d481b502f5d702e3/network-api/networkapi/wagtailpages/migrations/0047_update_searchimage_label_and_help_text.py)